packages/frontend-2/components/viewer/Controls.vue

```diff
@@ -297,6 +318,9 @@ const projectionShortcut = ref(
 const sectionBoxShortcut = ref(
   `Section Box (${getKeyboardShortcutTitle([ModifierKeys.AltOrOpt, 'b'])})`
 )
+const measureShortcut = ref(
+  `Measure Mode (${getKeyboardShortcutTitle([ModifierKeys.AltOrOpt, 'd'])})`
```

Comment:
Open for suggestions on a better keyboard shortcut for this. I chose D for "Dimensions"